<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="69610" started-at="2019-02-21T06:34:43Z" finished-at="2019-02-21T06:35:52Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="69610" started-at="2019-02-21T06:34:43Z" finished-at="2019-02-21T06:35:52Z">
      <class name="com.pharmeasy.MercuryUI.PurchaseEntry.TC020_VerifyThePurchaseEntryAfterSaveInPendingEntries">
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.pharmeasy.MercuryUI.PurchaseEntry.TC020_VerifyThePurchaseEntryAfterSaveInPendingEntries@15b3e5b]" name="beforeMethod" is-config="true" duration-ms="4849" started-at="2019-02-21T06:34:43Z" finished-at="2019-02-21T06:34:47Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.pharmeasy.MercuryUI.PurchaseEntry.TC020_VerifyThePurchaseEntryAfterSaveInPendingEntries.verifyThePurchaseEntryAfterSaveInPendingEntries() throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="verifyThePurchaseEntryAfterSaveInPendingEntries()[pri:0, instance:com.pharmeasy.MercuryUI.PurchaseEntry.TC020_VerifyThePurchaseEntryAfterSaveInPendingEntries@15b3e5b]" name="verifyThePurchaseEntryAfterSaveInPendingEntries" duration-ms="63866" started-at="2019-02-21T06:34:47Z" finished-at="2019-02-21T06:35:51Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.pharmeasy.MercuryUI.PurchaseEntry.TC020_VerifyThePurchaseEntryAfterSaveInPendingEntries.verifyThePurchaseEntryAfterSaveInPendingEntries(TC020_VerifyThePurchaseEntryAfterSaveInPendingEntries.java:69)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyThePurchaseEntryAfterSaveInPendingEntries -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.pharmeasy.MercuryUI.PurchaseEntry.TC020_VerifyThePurchaseEntryAfterSaveInPendingEntries@15b3e5b]" name="afterMethod" is-config="true" duration-ms="439" started-at="2019-02-21T06:35:51Z" finished-at="2019-02-21T06:35:52Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyThePurchaseEntryAfterSaveInPendingEntries status=FAILURE method=TC020_VerifyThePurchaseEntryAfterSaveInPendingEntries.verifyThePurchaseEntryAfterSaveInPendingEntries()[pri:0, instance:com.pharmeasy.MercuryUI.PurchaseEntry.TC020_VerifyThePurchaseEntryAfterSaveInPendingEntries@15b3e5b] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:com.pharmeasy.MercuryUI.PurchaseEntry.TC020_VerifyThePurchaseEntryAfterSaveInPendingEntries@15b3e5b]" name="endTest" is-config="true" duration-ms="420" started-at="2019-02-21T06:35:52Z" finished-at="2019-02-21T06:35:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
      </class> <!-- com.pharmeasy.MercuryUI.PurchaseEntry.TC020_VerifyThePurchaseEntryAfterSaveInPendingEntries -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
